pages/content/amp-dev/documentation/guides-and-tutorials/index.html

```diff
@@ -1,393 +1,228 @@
-    </div>
-    <h4 class="ap-o-card-grid-card-headline">Learn</h4>
-    <p class="ap-o-card-grid-card-copy">Get to know how AMP is different and acquire the fundamentals for how AMP works.</p>
-    <a href="{{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/email-spec/amp-email-format.md', locale=doc.locale).url.path}}"
-      class="ap-m-lnk">
-      <div class="ap-a-ico ap-m-lnk-icon">
-        {% do doc.icons.useIcon('icons/internal.svg') %}
-        <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#internal"></use></svg>
-      </div>
-      <span class="ap-m-lnk-text">AMPHTML email specification</span>
-    </a>
-  </div>
-  <div class="ap-o-card-grid-card">
-    <div class="ap-a-ico ap-o-card-grid-card-icon">
-      {% do doc.icons.useIcon('/icons/develop.svg') %}
-      <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#develop"></use></svg>
-    </div>
-    <h4 class="ap-o-card-grid-card-headline">Develop</h4>
-    <p class="ap-o-card-grid-card-copy">Create every content, layout, and feature your website needs.</p>
-    <a href="{{g.doc('/content/amp-dev/documentation/guides-and-tutorials/develop/style_and_layout/index.md', locale=doc.locale).url.path}}"
-      class="ap-m-lnk">
-      <div class="ap-a-ico ap-m-lnk-icon">
-        {% do doc.icons.useIcon('icons/internal.svg') %}
-        <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#internal"></use></svg>
-      </div>
-      <span class="ap-m-lnk-text">Style & layout</span>
-    </a>
-  </div>
-  <div class="ap-o-card-grid-card">
-    <div class="ap-a-ico ap-o-card-grid-card-icon ap-o-card-grid-card-icon-emails">
-      {% do doc.icons.useIcon('/icons/integration.svg') %}
-      <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#integration"></use></svg>
-    </div>
-    <h4 class="ap-o-card-grid-card-headline">Integrate</h4>
-    <p class="ap-o-card-grid-card-copy">Learn how to integrate AMP Email into your existing workflows.</p>
-    <a href="{{g.doc('/content/amp-dev/documentation/guides-and-tutorials/integrate/add-email.md', locale=doc.locale).url.path}}"
-      class="ap-m-lnk">
-      <div class="ap-a-ico ap-m-lnk-icon">
-        {% do doc.icons.useIcon('icons/internal.svg') %}
-        <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#internal"></use></svg>
-      </div>
-      <span class="ap-m-lnk-text">Add AMP to existing emails</span>
-    </a>
-  </div>
-  [% endif %]
+  {% with %}
+  {% set card_grid_items = [
+    {
+      "icon_name": "start",
+      "headline": "Start",
+      "text": "Building a 'Hello World' page and try adding your first interactive feature to an AMP page.",
+      "links": [
+        {
+          "url": g.doc('/content/amp-dev/documentation/guides-and-tutorials/start/create/basic_markup.md', locale=doc.locale).url.path,
+          "text": "Create your AMP HTML page"
```

Comment:
All of the documents in `card_grid_items` need to differ per selected format, see the current Guides & Tutorials overview page.